tests/test_ocr_runner_unit.py

```
@@ -0,0 +1,72 @@
+from scripts import ocr_runner
+
+
+class DummyProc:
+    def __init__(self, stdout=b'', returncode=0):
+        self.stdout = stdout
+        self.returncode = returncode
+
+
+def test_pdftotext_count(monkeypatch, tmp_path):
+    # monkeypatch subprocess.run to return a dummy stdout
+    def fake_run(*args, **kwargs):
```

Ruff (0.14.10)

Missing return type annotation for private function `fake_run`

(ANN202)